package.json

```diff
@@ -35,7 +35,7 @@
     "build": "yarn build:grunt",
     "build:tailwind": "tailwindcss -i ./assets/css/tailwind.css -o ./assets/generated/tailwind-output.css",
     "build:esbuild": "node .bin/esbuild.mjs",
-    "build:grunt": "grunt build",
+    "build:grunt": "grunt build && bash .bin/update-version.sh && grunt compress:main",
```

Comment:
I think we only get the updated version in build zip.
I am not sure but I think we always get and older version on Git repo because the updated version will be in build only and we push the older version.